srcalc/src/main/java/gov/va/med/srcalc/domain/HealthFactor.java

```diff
@@ -0,0 +1,64 @@
+package gov.va.med.srcalc.domain;
+
+import java.util.Objects;
+
+import org.joda.time.DateTime;
+
+/**
+ * A class that will store the name of a health factor along with the 
+ * date on which that factor was noted. Instances of this class are immutable.
+ */
+public class HealthFactor
+{
+    private final DateTime fDate;
+    private final String fName;
+    
+    public HealthFactor(final DateTime date, final String factor)
+    {
+        fDate = date;
+        fName = factor;
+    }
+    
+    /**
+     * Returns the date on which this health factor was noted.
+     */
+    public DateTime getDate()
```

Comment:
I think all you need is `LocalDate` here. I recommend reading the [Joda Time Quick Start Guide](http://www.joda.org/joda-time/quickstart.html).